android/app/src/main/java/org/fossasia/openevent/activities/auth/UserProfileActivity.java

```diff
@@ -122,18 +120,18 @@ protected void onDestroy() {
         super.onDestroy();
         if (swipeRefreshLayout != null)
             swipeRefreshLayout.setOnRefreshListener(null);
+    }
+    private void getUser() {
```

Comment:
Add a space above and rename to subscribeUser

Reply:
Oh i refactored other function inside ViewModel ! I will make the changes right away